{"uuid":"0990f540-f8cb-462f-931c-13af16ad2974","historyId":"fef12dcf14351656b1caac3f42149d64","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testrunners.MyTestRunner"},{"name":"testClass","value":"testrunners.MyTestRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testrunners.MyTestRunner"},{"name":"host","value":"MacBook-Pro.attlocal.net"},{"name":"thread","value":"11077@MacBook-Pro.attlocal.net.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Successful Submission] but found [Error occurred during submission]","trace":"java.lang.AssertionError: expected [Successful Submission] but found [Error occurred during submission]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:656)\n\tat org.testng.Assert.assertEquals(Assert.java:666)\n\tat com.stepdefinitions.ArraySteps.the_user_should_be_presented_with_successful_submission_message_on_question_two(ArraySteps.java:489)\n\tat âœ½.The user should be presented with successful submission message on question two(file:///Users/Roops/git/DS_Algo/src/test/resources/features/FArray.feature:176)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Array\""},{"name":"arg0","value":"\"The user is able to run and submit code in tryEditor for Max Consecutive Ones\""}],"start":1691241827061,"stop":1691241830801}